src/main/resources/application.yml

```diff
@@ -17,14 +17,7 @@ spring:
     serialization:
       WRITE_DATES_AS_TIMESTAMPS: false
 
-  security:
-    oauth2:
-      resourceserver:
-        jwt:
-          issuer-uri: ${oauth.endpoint.url}/issuer
-          jwk-set-uri: ${oauth.endpoint.url}/.well-known/jwks.json
-
-oauth.endpoint.url: http://localhost:8080/auth
+oauth.endpoint.url: http://localhost:9090/auth
```

Comment:
don't see why this is required?

Reply:
Just because if you're running locally against an Oauth server you don't want it to run on the same port as elite2.  That said, by default it's not used.  Shall I get rid?